File: src/main/java/ru/abe/slaves/potrebot/PotrebotApplication.java
package ru.abe.slaves.potrebot;

import lombok.RequiredArgsConstructor;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@EnableAsync
@SpringBootApplication
@RequiredArgsConstructor
public class PotrebotApplication {
    public static void main(String[] args) {
        SpringApplication.run(PotrebotApplication.class, args);
    }
}

File: src/main/java/ru/abe/slaves/potrebot/PotrebotApplication.kt
package ru.abe.slaves.potrebot

import lombok.RequiredArgsConstructor
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@RequiredArgsConstructor
open class PotrebotApplication

fun main(args: Array<String>) {
    runApplication<PotrebotApplication>(*args)
}
